components/hackathon/common/PrizeSection.tsx

```diff
@@ -0,0 +1,43 @@
+import { Center, Divider, Flex } from "@chakra-ui/react";
+import React from "react";
+import { Heading, Text } from "tw-components";
+
+interface IPrizeSection {
+  prizes: {
+    title: string;
+    prize: string;
+  }[];
+}
```

Comment:
this should be `PrizeSectionProps` based on how we generally name props types

also I would recommend extracting `Prize` as a type here so then it becomes `prizes: Prize[]` **or** alternatively using the `Array<{...}` syntax to make this more readable